app/policies/vaccination_record_policy.rb

```diff
@@ -19,7 +19,7 @@ def update?
   end
 
   def destroy?
-    user.is_superuser?
+    user.is_superuser? && record.source != "nhs_immunisations_api"
```

Comment:
```suggestion
    user.is_superuser? && !record.nhs_immunisations_api?
```

It might be worth adding a `prefix` to the enum (perhaps `sourced_from`), then we could do something like:

```rb
record.sourced_from_nhs_immunisations_api?
```